src/element/textWysiwyg.tsx

```diff
@@ -46,7 +52,7 @@ export function textWysiwyg({
     if (ev.key === KEYS.ESCAPE) {
       ev.preventDefault();
       if (initText) {
-        editable.innerText = initText;
+        editable.innerText = trimText(initText);
```

Comment:
AFAIK no need to do it here since we do it in `handleSubmit`

Reply:
Yeah, you're right!